src/traces/scattermapbox/attributes.js

```diff
@@ -26,6 +26,58 @@ module.exports = overrideAll({
     lon: scatterGeoAttrs.lon,
     lat: scatterGeoAttrs.lat,
 
+    cluster: {
+        enabled: {
+            valType: 'boolean',
+            role: 'info',
+            dflt: false,
+            editType: 'plot',
+            description: 'Determines whether clustering is enabled or disabled.'
+        },
+        maxzoom: {
+            valType: 'number',
+            role: 'info',
+            min: 0,
+            max: 22,
+            dflt: 14,
+            description: 'Sets the maximum zoom level for the cluster.',
+        },
+        radius: {
+            role: 'info',
+            valType: 'number',
+            dflt: 50,
+            description: 'Radius of each cluster when clustering points.',
+        },
+        step: {
+            role: 'info',
+            valType: 'number',
+            arrayOk: true,
+            dflt: -1,
+            min: -1,
+            description: [
+                'Sets the steps for each cluster.'
+            ].join(' ')
+        },
+        size: {
+            role: 'info',
+            valType: 'number',
+            arrayOk: true,
+            dflt: 20,
```

Comment:
Here is an example illustrating why the cluster need a size attribute different from the markers. The marker size sets the size of the markers when the cluster dissolves. The cluster size attributes determines the size of the clusters.

In the example posted below, it means that if there is more than 3 points in the cluster the cluster will have a size of 40. If the there is fewer than 3, it will have the size 20.

https://codepen.io/elben10/pen/yLJRpYQ